functions/tests/integration/alexa/dialog.yaml

```diff
@@ -45,8 +45,18 @@
       intend: AMAZON.we_definitely_do_not_handle_it
     assistant: "Sorry, I'm afraid I don't follow you."
 
-- scenario: "failed request for a collection from IA should gracefully fail"
+- scenario: "failed request for a collection from IA should gracefully fail in one go intent"
   dialog:
   - user: 'play turbo folk'
     assistant: 'We experience some technical problems on IA server,
                 please try again later or try something else.'
+
+- scenario: "failed request for a collection from IA should gracefully fail in music query intent"
+  dialog:
+  - user: 'seventy eighties'
+    assistant: 'You''ve selected 78s.
+                What genre of music would you like to listen to?
+                Please select a genre like Jazz, Instrumental or Dance?'
+  - user: 'turbo folk music'
+    assistant: 'We experience some technical problems on IA server,
```

Comment:
Code looks great Eugene! I would change the last assistant line from `We experience some technical problems on IA server` to `We are currently experiencing some technical difficulties on the Archive server. Please try again later or try saying something else.` Sounds more natural!